Cython/Compiler/Nodes.py

```diff
@@ -2319,7 +2319,8 @@ class CFuncDefNode(FuncDefNode):
     #  is_static_method whether this is a static method
     #  is_c_class_method whether this is a cclass method
 
-    child_attrs = ["base_type", "declarator", "body", "py_func_stat"]
+    child_attrs = ["base_type", "declarator", "body", "py_func_stat", "decorators"]
+    outer_attrs = ["decorators"]
```

Comment:
It is not clear to me how these are used, and if I need to add `'decorators'` to these.

Reply:
thanks